get-started/the-stack.md

```diff
@@ -7,97 +7,147 @@ mapped_pages:
+If you want to transform and enrich data before it's stored, you can use {{es}} ingest pipelines or {{ls}}.
 
-### Consume [_consume]
+Trying to decide which ingest components to use? Refer to [](/manage-data/ingest.md) and [](/manage-data/ingest/tools.md).
 
-Use {{kib}} to query and visualize the data that’s stored in {{es}}. Or, use the {{es}} clients to access data in {{es}} directly from common programming languages.
+### {{agent}} and {{integrations}}[stack-components-agent]
 
-#### {{kib}} [stack-components-kibana]
+{{agent}} is a single, unified way to add monitoring for logs, metrics, and other types of data to a host.
+It can also protect hosts from security threats, query data from operating systems, and forward data from remote services or hardware.
+Each agent has a single policy to which you can add [integrations](integration-docs://reference/index.md) for new data sources, security protections, and more.
+You can also use [{{agent}} processors](/reference/fleet/agent-processors.md) to sanitize or enrich your data.
 
-{{kib}} is the tool to harness your {{es}} data and to manage the {{stack}}. Use it to analyze and visualize the data that’s stored in {{es}}. {{kib}} is also the home for the Search, Observability and Security solutions. [Learn more about {{kib}}](/explore-analyze/index.md).
+{{fleet}} enables you to centrally manage {{agents}} and their policies.
+Use {{fleet}} to monitor the state of all your {{agents}}, manage agent policies, and upgrade {{agent}} binaries or integrations.
 
-#### {{es}} clients [stack-components-elasticsearch-clients]
+[Learn more about {{agent}}](/reference/fleet/index.md).
 
-The clients provide a convenient mechanism to manage API requests and responses to and from {{es}} from popular languages such as Java, Ruby, Go, Python, and others. Both official and community contributed clients are available. [Learn more about the {{es}} clients](/reference/elasticsearch-clients/index.md).
+### {{product.apm}} [stack-components-apm]
 
-## Version compatibility
-```{applies_to}
-deployment:
-  self:
-```
+{{product.apm}} is an application performance monitoring system.
+It allows you to monitor software services and applications in real-time by collecting detailed performance information on response time for incoming requests, database queries, calls to caches, external HTTP requests, and more.
+This makes it easy to pinpoint and fix performance problems quickly.
 
-:::{include} /deploy-manage/deploy/_snippets/stack-version-compatibility.md
-:::
+[Learn more about {{product.apm}}](/solutions/observability/apm/index.md).
+
+### OpenTelemetry Collector [stack-components-otel]
+
+[OpenTelemetry](https://opentelemetry.io/docs)(OTel) is a vendor-neutral observability framework for collecting, processing, and exporting telemetry data. Elastic is a member of the Cloud Native Computing Foundation (CNCF) and active contributor to the OpenTelemetry project.
 
-## Installation order
```

Comment:
same with this - can't remove and not relocate

Reply:
Ditto it's a snippet (https://github.com/elastic/docs-content/blob/main/deploy-manage/deploy/_snippets/installation-order.md) that currently exists in these other pages:

- https://www.elastic.co/docs/deploy-manage/deploy/self-managed/installing-elasticsearch#installation-order
- https://www.elastic.co/docs/deploy-manage/deploy/self-managed/install-kibana#installation-order

Reply:
I tend to agree with Lisa on this one, feels out of place here TBH

Reply:
the longer-term fix for this is baking this info into the install guides for all products ... easier said than done. I see where you're coming from though